lib/dash/dash_parser.js

```diff
@@ -998,15 +1003,22 @@ shaka.dash.DashParser.prototype.createFrame_ = function(
   parent = parent || /** @type {shaka.dash.DashParser.InheritanceFrame} */ ({
     contentType: '',
     mimeType: '',
-    codecs: ''
+    codecs: '',
+    frameRate: undefined
   });
   baseUris = baseUris || parent.baseUris;
 
   var parseNumber = XmlUtils.parseNonNegativeInt;
+  var parseFloat = XmlUtils.parseFloat;
   var uris = XmlUtils.findChildren(elem, 'BaseURL').map(XmlUtils.getContents);
 
   var contentType = elem.getAttribute('contentType') || parent.contentType;
   var mimeType = elem.getAttribute('mimeType') || parent.mimeType;
+  var frameRate = XmlUtils.parseAttr(elem, 'frameRate',
+      function(expr) {
+        // frameRate can be an expression e.g. "1000000/42000"
+        return parseFloat(String(eval(expr)));
```

Comment:
Oh, of course. Very sorry about this. Don't know what I was thinking. Will fix this of course